portal-kernel/src/com/liferay/portal/kernel/model/Company.java

```diff
@@ -101,12 +101,6 @@ public String getShortName()
 
 	public boolean isAutoLogin();
 
-	/**
```

Comment:
@jeffwu0724 Company and CompanyWrapper are generated classes, usually we did not modify them manually. I will accept the code this time as it is a small change. Next time, please modify CompanyImpl, remove the method there, then run build-service to generate the change on Company and CompanyWrapper